### DISCUSSION TITLE
Site Monitoring will not turn off ?

### DISCUSSION THREAD

**Discussion by kentmorrison** [akeeba/panopticon#612] [Q&A (Community Support)]
I have 4 sites successfully connected to Panopticon, and properly updating the site info, versions, etc.
VERY cool product !
I read the entire section of the documentation referrring to Site Monitoring, and wanted to try it out.
When I had a few different problems with using SIte Monitoring, I decided to turn it off.
Now that I have turned off Site Monitoring, I am still getting email notificaitions about the uptime status of several sites, so it seems that Site Monitoring has not been disabled.
Is there some other way to disable SIte monitoring than the menu item in System Configuration ?
Thanks for your time !
Kent

**Comment by nikosdion** [comment]
Yes, there is a way.

Administration, System Configuration, Automatic Tasks.

Find the "Uptime Monitoring" section and set "Uptime Monitoring Provider" to None.

Then, click on Save & Close.

**Comment by kentmorrison** [comment]
Nicholas,
That is what I am telling you, the "Set Uptime Monitoring Provider" to none had no effect on stopping the email notifications of site downtime.
I got at least 50 or 60 such notifications in the hours after setting to "none".

Last night I turned off all email notification also, and the flood of emails stopped.
I just turned "Send Email" back on, and so far, no email notifications about downtime.

I was careful to do these tasks in the correct order, and I am trying my best to help you determine if this is really a bug.

I really want to use Panopticon, I have some experience with beta test of software, and I am glad to help in any way.

Would you like me to gather more info about the matter ?
Do you have suggestions about the best way to proceed ?

Regards,
Kent

​

Kent Morrison

Ponca Project Communications Director

Email :
***@***.***

Website:
http://PoncaProject.com
Telephone:   970 756 0040

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by nikosdion** [reply]
Your mail server backlogged the emails and was sending them with a delay. Nothing to do with us.

**Comment by kentmorrison** [comment]
Thanks Nicholas !

​

Kent Morrison

Ponca Project Communications Director

Email :
***@***.***

Website:
http://PoncaProject.com
Telephone:   970 756 0040

Sent with [Proton Mail](https://proton.me/) secure email.
